client/src/api/BackendAccess.tsx

```diff
@@ -1,8 +1,16 @@
-import axios from 'axios';
+import axios from "axios";
+import { useRecoilValue } from "recoil";
+import { userBearerToken } from "../AppState";
 
-const API_URL = 'https://localhost:44382/api';
+const API_URL = process.env.REACT_APP_APIBASEURL as string;
```

Comment:
Aha, so würde man das sauber lösen :-)

Reply:
Damit es dann auch auf der deployten Version in Azure läuft.